docs/readme.md

```diff
@@ -0,0 +1,49 @@
+# Documentation
+
+## Where do I start?
+
+The documentation here is comprehensive, so it may be hard for someone new to know where to start reading to understand how the tool works or how it is architected. The best starting point depends on what your end goal is:
+
+- If you are interested in using the tool to run some tests, there are a few documents you need to read, roughly in the following order:
+    - [How-to guide](load-test-how-to-use.md): this guide is a great overview of how things work, as well as tips and tricks that will make your life easier. It also links to multiple other documents, which you can follow to broaden your understanding. Read it completely first before running any tests!
+    - [How to run a load-test locally](local_loadtest.md): a specific guide to start running load-tests locally in your computer, no AWS needed.
+    - [How to run a load-test in Terraform](terraform_loadtest.md): a more advanced guide to deploy a cluster to AWS to run a more powerful load-test.
```

Comment:
Can we add a note that this is the recommended way to do any serious load-test aiming at production?

Reply:
Done in https://github.com/mattermost/mattermost-load-test-ng/pull/612/commits/35e6b2f4640b5c000b0e32def7f5439305594278